sdcm/mgmt/cli.py

```diff
@@ -923,7 +923,7 @@ def __init__(self, manager_node):
         LOGGER.info("Initiating Scylla-Manager, version: {}".format(self.sctool.version))
         list_supported_distros = [Distro.CENTOS7,
                                   Distro.ROCKY8, Distro.ROCKY9,
-                                  Distro.DEBIAN10, Distro.DEBIAN11,
+                                  Distro.DEBIAN11, Distro.DEBIAN12,
```

Comment:
Damn, I've missed this silly list in my PR.

We need to remove this check as well, we should support any debian or rhel based distro, no need for extra checks

Reply:
Replaced with a simple `is_blank_like` check.